screenpy/actor.py

```diff
@@ -13,6 +13,8 @@
 from .protocols import Forgettable, Performable
 from .speech_tools import get_additive_description
 
+# pylint: disable=too-many-public-methods
```

Comment:
I'm not sure why `pylint` was OK with all the aliases when they were created using `=` but complains now, but this sure shut it up.

Reply:
The aliases within `_TimeframeBuilder` should get converted also.
`millisecond = milliseconds`

Reply:
I didn't convert them because they aren't surfaced for documentation (currently) and also do not have the problems that the `classmethod`s did. 

... Unless they do?

Reply:
Oh, you're right. They are just aliases.  AFAICT they don't have the same problem as the classmethods.

Reply:
🤫